controller/src/main/java/io/pravega/controller/server/eventProcessor/AbortRequestHandler.java

```diff
@@ -79,15 +80,15 @@ public AbortEventProcessor(final StreamMetadataStore streamMetadataStore,
     }
 
     @Override
-    protected void process(AbortEvent event, Position position) {
+    protected CompletableFuture<Void> processEvent(AbortEvent event, EventProcessor.Writer<AbortEvent> writer) {
```

Comment:
This writer is actually not used anywhere in this call, why do we have it? Is it simply because of the signature being overridden?

Reply:
the writer is used because commitRequestHandler to postpone a commit in case old epoch is not deleted...
Right now i have kept is simple and added it in the signature here so that all request handlers get a writer, and they can choose to use it or not in their processing. 

As part of my clean up in 1738 i am already cleaning this up to remove the writer from here.